README.md

```diff
@@ -1,17 +1,17 @@
 # scale-encode
 
 `parity-scale-codec` provides an `Encode` trait which allows types to SCALE encode themselves based on their shape.
-This crate builds on this, and allows types to encode themselves based on `scale_info` type information. It
-exposes two traits:
+This crate builds on this, and allows types to encode themselves based on type information from a [`TypeResolver`]
```

Comment:
does the link [`TypeResolver`] really work in the readme?

Reply:
Ooh good point; I copied it from the lib README again to sync it all up and forgot to remove the []'s :)